packages/js/src/components/WincherPerformanceReport.js

```diff
@@ -16,7 +16,7 @@ import { Alert, NewButton } from "@yoast/components";
 import WincherNoTrackedKeyphrasesAlert from "./modals/WincherNoTrackedKeyphrasesAlert";
 import { getKeyphrasePosition, PositionOverTimeChart } from "./WincherTableRow";
 import WincherReconnectAlert from "./modals/WincherReconnectAlert";
-import WincherUpgradeCallout from "./modals/WincherUpgradeCallout";
+import WincherUpgradeCallout, { useTrackingInfo } from "./modals/WincherUpgradeCallout";
```

Comment:
Seems sorta strange we're importing `useTrackingInfo` from `WincherUpgradeCallout` only to call it and pass it back into the `WincherUpgradeCallout`

Reply:
We need it outisde `WincherUpgradeCallou` since we use it in `WincherSeoPerformance` for both `WincherUpgradeCallou` (line 486) and `TableContent` (line 500).